### PR TITLE
New train

### PR DESCRIPTION
**Pull request by Jacobsolawetz**

### CODE REVIEW

**Comment by Jacobsolawetz** [comment]
There is nothing significant here other than allowing for a developer to develop this against api.staging.roboflow.com and bumping the npm package number that we published

**Comment by Jacobsolawetz** [comment]
Brad's got meetings so going to auto merge to push new offline improvements on top, will leave branch alive for review after

File: server/index.js [hunk 1]
```diff
@@ -253,6 +256,7 @@ app.post(
     transformImageBody,
 	function(req, res, next) {
 		req.newFormat = true;
+        req.staging = staging;
```
Comment (yeldarby) [inline server/index.js:259]: Could you fix the indentation on this

Comment (Jacobsolawetz) [reply server/index.js:259]: My IDE is messed up on some dimension 
image

Comment (Jacobsolawetz) [reply server/index.js:259]: Fixed in outstanding PR https://github.com/roboflow-ai/inference-server/commit/287f26ada6a86658c568bd03af4fa53acc327839

**Pull request merged**